velox/functions/sparksql/tests/StringTest.cpp

```diff
@@ -448,6 +449,19 @@ TEST_F(StringTest, overlayVarbinary) {
   EXPECT_EQ(overlay("Spark SQL", "##", -10, 4), "##rk SQL");
 }
 
+TEST_F(StringTest, repeat) {
+  const auto stringRepeat = [&](const std::optional<std::string>& str,
+                                const std::optional<int32_t>& times) {
+    return evaluateOnce<std::string>("repeat(c0, c1)", str, times);
+  };
+
+  EXPECT_EQ(stringRepeat("hh", 2), "hhhh");
+  EXPECT_EQ(stringRepeat("abab", 0), "");
+  EXPECT_EQ(stringRepeat("123\u6570", 2), "123\u6570123\u6570");
+  VELOX_ASSERT_USER_THROW(
+      stringRepeat("hh", 10001), "Repeat times is too big.");
```

Comment:
Can we add tests for negative times and empty input string?

Reply:
Added, thanks